composer.json

```diff
@@ -17,7 +17,7 @@
     "require-dev": {
         "codeigniter/coding-standard": "^1.1",
         "fakerphp/faker": "^1.9",
-        "friendsofphp/php-cs-fixer": "3.4.*",
+        "friendsofphp/php-cs-fixer": "^3.6",
```

Comment:
Do we also update `admin/framework/composer.json` ?

Reply:
Good catch! I'll update in a while.

Reply:
Added.